### DISCUSSION TITLE
Setting Solver Verbosity

### DISCUSSION THREAD

**Discussion by jusevitch-mitll** [ds4dm/ecole#255] [Questions]
What's the best way to make the underlying SCIP instance verbose when using Ecole?

I tried adding the entry `"display/verblevel": 4` to the `scip_params` dictionary when setting up a ([Configuring](https://doc.ecole.ai/py/en/stable/reference/environments.html#ecole.environment.Configuring)) Ecole environment, but I'm not getting any display output when the SCIP solver is called.

**Comment by AntoinePrv** [comment]
In Ecole we call [`SCIPmessagehdlrSetQuiet`](https://www.scipopt.org/doc-7.0.3/html/message_8c.php#ab25bb4a84472d441fa59625586a75e57), but I see no way to restore it through Ecole's Python interface or PySCIPOpt.

This would need to be added.
